common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -484,6 +575,70 @@ public void putValueInSuccessEvent(final @NonNull ApiEndEvent event, final @NonN
                 });
     }
 
+    /**
+     * Get the intent for the broker Account Transfer request
+     *
+     * @param parameters a {@link AccountTransferTokenCommandParameters}
+     * @return an {@link Intent} for initiating Broker Account Transfer interactive activity.
+     */
+    @NonNull
+    private Intent getBrokerAuthorizationIntentForAccountTransfer(
+            final @NonNull AccountTransferTokenCommandParameters parameters) throws BaseException {
+        return getBrokerOperationExecutor().execute(parameters,
+                new BrokerOperation<Intent>() {
+                    private String negotiatedBrokerProtocolVersion;
+
+                    @Override
+                    public void performPrerequisites(final @NonNull IIpcStrategy strategy) throws BaseException {
+                        verifyTokenParametersAreSupported(parameters);
+                        negotiatedBrokerProtocolVersion = hello(strategy, parameters.getRequiredBrokerProtocolVersion());
```

Comment:
if the negotiatedBrokerProtocolVersion is lower than the new version you just added, it means the broker does NOT support this operation.

In that case, you might want to fail the request early. (no need to make the request to the broker because it'll just throw "not supported" back)

Reply:
So, in the parameters for ATv2, OneAuth will send in 16.0 protocol version. wouldn't this already take care of this issue? We'd get an error result from the hello operation in that case, right?

Reply:
No longer applicable